conda_recipes/maya-2025/README.md

```diff
@@ -0,0 +1,76 @@
+# Maya 2025 conda build recipe
+
+## Instructions for Maya plugin packages
+
+This Maya conda build recipe configures the `MAYA_MODULE_PATH` environment variable
+to include several paths to search for plugin `.mod` files. When creating a plugin
+package, place its `.mod` file in one of these so that Maya loads the plugin at startup.
+
+* `$PREFIX/usr/autodesk/maya$MAYA_VERSION/modules`
+* `$PREFIX/usr/autodesk/modules/maya/$MAYA_VERSION`
+* `$PREFIX/usr/autodesk/modules/maya`
+
+## Download the archive file for Linux
+
+Download the Autodesk_Maya_2025_Linux_64bit.tgz full download file from Autodesk, and
+place it in the `conda_recipes/archive_files` directory in your git clone of the
+[https://github.com/aws-deadline/deadline-cloud-samples](deadline-cloud-samples) repository for
+submitting package build jobs,
+
+## Creating an archive file for Windows
+
+The Windows installer requires Administrator permissions that are not available in most conda package
+build environments, such as on a Deadline Cloud service-managed fleets. Follow these instructions to
+install Maya 2025 on a freshly created EC2 instance as Administrator, and create an archive file
+for use by the conda build recipe.
+
+1. Launch a fresh Windows Server 2022 instance.
+    1. From the AWS EC2 management console, select the option to Launch instance.
+    2. Enter instance name "Create Windows Maya archive".
+    3. Select "Microsoft Windows Server 2022 Base" for the AMI. (As of writing, Windows Server 2025 did not work with these instructions.)
+    4. Select an instance type with enough vCPUs and RAM, for example c5.4xlarge has 8 vCPUs and 16 GiB RAM.
+    5. Select "Proceed without a key pair" for the "Key pair (login)" option.
+    6. We will use SSM port forwarding to avoid sending RDP protocol traffic directly over the internet.
+        1. Make sure that "Allow RDP traffic" is unchecked.
```

Comment:
why do we not allow RDP traffic here if we connect with RDP in step 2?

Reply:
Because we're documenting how to set it up with SSM port forwarding that sends the traffic through an encrypted tunnel instead of over the internet.